src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Resources/Resources.resx

```diff
@@ -157,6 +157,10 @@ This test may, or may not be the source of the crash.</value>
   <data name="ProcDumpEnvVarEmpty" xml:space="preserve">
     <value>Required environment variable PROCDUMP_PATH was null or empty. Set PROCDUMP_PATH to path of folder containing appropriate procdump executable.</value>
   </data>
+  <data name="ProcDumpNotFound" xml:space="preserve">
+    <value>{0} could not be found, please make sure that the executable is available on PATH. Alternatively set PROCDUMP_PATH environment variable to a directory that contains {0} executable</value>
```

Comment:
Not sure if we have any preference, in some other repos, we were usually putting paths/input values etc between quotes (single quotes).

Reply:
It is just a file name, no quotes is imho fine.